packages/client/src/assets/fonts/minecraft/minecraft.css

```
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'Minecraft';
+  font-style: normal;
+  font-weight: 400;
+  src: url('minecraft-regular.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Minecraft';
```

Тут наверное уже надо писать 
```suggestion
  font-family: 'MinecraftBold';
```

---

Нет, здесь все правильно.
Задано семейство Minecraft. Используется так:
`ctx.font = '400 24px Minecraft'` начертание regular
`ctx.font = '700 40px Minecraft'` начертание bold